manifests/application.pp

```diff
@@ -55,7 +55,7 @@
   $enable_deb = $katello::params::enable_deb
   $pulp_url = $katello::params::pulp_url
   $pulp_ca_cert = $certs::katello_server_ca_cert # TODO: certs::apache::...
-  $candlepin_url = $katello::params::candlepin_url
+  $candlepin_url = "https://${katello::candlepin_host}:8443/candlepin"
```

Comment:
The `katello::application` should not assume the `katello` class exists. I should write a PR, but after 2.1 I want to deprecate it altogether. Anything that's shared between `katello::application` and `katello::candlepin` should go through `katello::params` (as the URL was previously). You can then remove the parameter from `init.pp`

Reply:
I think I follow what you are suggesting. So check out this line and change now... and let me know if I got there or not.